packages/e2e/README.md

```diff
@@ -1,195 +1,3 @@
 # e2e (deprecated)
 
-## DEPRECATED
-
```

Comment:
This package is several deprecations old, and only this README exists in the tree (in case people have links to the package), so rather than do a targeted update I just nuked everything but the pointer.